src/app/core/services/storage/storage.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+import { enableAkitaProdMode, persistState } from '@datorama/akita';
+import { environment } from '../../../../environments/environment.prod';
```

Comment:
Environment imports should be from `'../../../../environments/environment`.

Reply:
Fixed